sentry-types/src/protocol/v7.rs

```diff
@@ -1454,3 +1501,96 @@ impl<'a> fmt::Display for Event<'a> {
         write!(f, "Event(id: {}, ts: {})", self.event_id, self.timestamp)
     }
 }
+
+mod span {
+    use super::*;
+
+    pub fn default_id() -> Uuid {
+        Uuid::new_v4()
+    }
+
+    pub fn default_timestamp() -> DateTime<Utc> {
+        Utc::now()
+    }
+}
+
+/// Represents a tracing span.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct Span {
+    #[serde(default = "event::default_id", serialize_with = "event::serialize_id")]
+    pub event_id: Uuid,
+    #[serde(default = "event::default_id", serialize_with = "event::serialize_id")]
```

Comment:
```suggestion
    #[serde(default = "span::default_id", serialize_with = "event::serialize_id")]
```
or rather remove the separate module?